_clients/dot-net.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: .NET clients
+nav_order: 75
+has_children: true
+has_toc: false
+---
+
+# .NET clients
+
+OpenSearch has two .NET clients: a low-level [OpenSearch.Net]({{site.url}}{{site.baseurl}}/clients/OpenSearch-dot-net/) client, and a high-level [OpenSearch.Client]({{site.url}}{{site.baseurl}}/clients/OSC-dot-net/) client.
+
+[OpenSearch.Net]({{site.url}}{{site.baseurl}}/clients/OpenSearch-dot-net/) is a low-level .NET client that provides the foundational layer of communicating with OpenSearch. It is dependency-free, and it can handle round-robin load balancing, transport, and the basic request/response cycle. OpenSearch.Net contains methods for all OpenSearch API endpoints.
```

Comment:
"... of communication with OpenSearch"? Maybe "...of communicating with..." is conventional in this space. But caught my eye.